docs/docs/dev_docs/tutorials/token_portal/withdrawing_to_l1.md

```diff
@@ -53,13 +55,17 @@ We call this pattern _designed caller_ which enables a new paradigm **where we c
 
 Before we can compile and use the contract, we need to add two additional functions.
 
-We need a function that let's us read the token value.
+We need a function that lets us read the token value. Paste this into `main.nr`:
 
 #include_code read_token /yarn-project/noir-contracts/contracts/token_bridge_contract/src/main.nr rust
 
-And the `compute_note_hash_and_nullifier` required on every contract.
+And the `compute_note_hash_and_nullifier` required on every contract:
+
+```rust
+#include_code compute_note_hash_and_nullifier_placeholder /yarn-project/noir-contracts/contracts/token_bridge_contract/src/main.nr raw
+}
```

Comment:
Same comment as above, should we link to source since wrapping this seems to remove it?

Reply:
same answer